### PR TITLE
CAL-269 moved documentation out of documentation profile

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Eliminates the documentation profile from the `distribution/docs` and `distribution/ddf` pom.xml files. 

#### Who is reviewing it?

@mcalcote @vinamartin @Lambeaux @jrnorth @oconnormi 

#### Select at least one member from relevant component team(s) from below.

[Build](https://github.com/orgs/codice/teams/build)
[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@coyotesqrl
@figliold
@lessarderic
@pklinef
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

- Run `mvn clean` from top level directory (optional, but helps confirm whether module is running or not.)
- build on a clean repo with the command `mvn clean install -pl \!distribution/docs`. Expected results: no `target` directory created in distribution/docs module; download of previous snapshot artifacts into repository for `ddf/docs`
- unzip distribution zip. The `documentation` directory should have a full set of documentation html and pdf. Open the `documentation.html` file, scroll to the very bottom of the page and check the _last updated_ date.
- build again without the `-pl` flag. (`mvn clean install`). Expected results: target directory created in `distribution/docs` with complete artifacts, newly built artifacts in the repository.

#### Any background context you want to provide?

"Skipping" documentation by disabling the profile still causes an incomplete selection of artifacts to be generated, which can cause unexpected behavior if: a) a following build does not 'clean', b) during incremental builds. 

#### What are the relevant tickets?
[CAL-269](https://codice.atlassian.net/browse/CAL-269)
#### Screenshots (if appropriate)
n/a
#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/alliance/pom.xml [hunk 1]
```diff
@@ -395,58 +395,41 @@ org.codice.alliance.corba_default_port=2809</concat>
...
+                        they were declared in.
+
+                        See https://issues.apache.org/jira/browse/MNG-5799 as well
+                        as the related tickets for more details.
+                        -->
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
```
Comment (lessarderic) [inline distribution/alliance/pom.xml]: ❓ Do we need to add a dependency on the `org.codice.alliance:docs` artifact in the `<dependencies>` section?

File: distribution/docs/pom.xml [hunk 2]
```diff
@@ -45,308 +45,285 @@
...
-            <dependencies>
-                <dependency>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-dependency-plugin</artifactId>
+                    <version>2.10</version>
+                    <executions>
+                        <execution>
+                            <id>unpack</id>
+                            <phase>validate</phase>
```
Comment (lessarderic) [inline distribution/docs/pom.xml]: ❓ For the `unpack`, `copy-adocs` and `copy-images`, would it make more sense to use the `generate-resources` or `process-resources` phase instead?

File: distribution/docs/pom.xml [hunk 3]
```diff
@@ -45,308 +45,285 @@
...
+                            <goals>
+                                <goal>unpack</goal>
+                            </goals>
+                            <configuration>
+                                <artifactItems>
+                                    <artifactItem>
```
Comment (lessarderic) [inline distribution/docs/pom.xml]: ❓ Do we need to add dependencies on these artifacts in the `<dependencies>` section?

File: distribution/alliance/pom.xml (hunk 1 above)
Comment (ricklarsen) [inline distribution/alliance/pom.xml]: done.

File: distribution/docs/pom.xml (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/pom.xml]: done.

File: distribution/docs/pom.xml (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/pom.xml]: fixed.

**Comment by Lambeaux** [comment]
Hero'd and works as expected 👍

File: distribution/docs/pom.xml [hunk 4]
```diff
@@ -24,6 +24,37 @@
     <properties>
         <skipTests>true</skipTests>
     </properties>
+        <dependencies>
+        <dependency>
```
Comment (coyotesqrl) [inline distribution/docs/pom.xml]: ❓ Is this dependency required? I'd think you'd only need the classified one below.

File: distribution/docs/pom.xml [hunk 5]
```diff
@@ -45,308 +76,268 @@
...
+                            <resources>
+                                <resource>
+                                    <directory>${project.build.directory}/images/ddf/docs-${ddf.version}</directory>
+                                    <filtering>false</filtering>
+                                    <includes>
+                                        <include>*</include>
+                                    </includes>
+                                </resource>
+                                <resource>
+                                    <directory>${project.build.directory}/images/admin/ docs-${ddf.version}</directory>
```
Comment (coyotesqrl) [inline distribution/docs/pom.xml]: Remove space after `admin/`

File: distribution/docs/pom.xml (hunk 4 above)
Comment (ricklarsen) [inline distribution/docs/pom.xml]: fixed.

File: distribution/docs/pom.xml (hunk 5 above)
Comment (ricklarsen) [inline distribution/docs/pom.xml]: fixed.

**Pull request merged**